template/vars/default_community.yaml

```diff
@@ -1,3 +1,3 @@
 env_vars:
   NUM_LOOPBACK_AUDIO_DEVICES: 32
-  NUM_LOOPBACK_VIDEO_DEVICES: 0
+  NUM_LOOPBACK_VIDEO_DEVICES: 1
```

Comment:
Maybe worth a note that at least at present this is only used for docker-worker's own CI.